### DISCUSSION TITLE
Version 23.02 - Copy device http error 500

### DISCUSSION THREAD

**Discussion by mfenton01** [opendcim/openDCIM#1432] [Q&A] [answered]
Working through 23.02 upgrade when trying to copy a device getting the following http 500 error.. Not sure if its DCIM or Chrome . Not showing any error in the http /error log .

![Snag_12e3db7](https://github.com/opendcim/openDCIM/assets/2854433/b1815ae3-9f1f-49f1-92c3-7d1e68e8aa29)

**Comment by wilpig** [comment, marked as answer]
Issue created.
https://github.com/opendcim/openDCIM/issues/1433
